tests/e2e/apiserver_test.go

```diff
@@ -41,31 +38,27 @@ const (
 	testTimeout  = 300 * time.Second
 )
 
-var _ = Describe("[apiserver] [Disruptive] the apiserver", func() {
-	var (
-		cs  clientset.Interface
-		cfg *restclientset.Config
-	)
+var SIGDescribe = framework.SIGDescribe("api-machinery")
+
+var _ = SIGDescribe("apiserver", framework.WithDisruptive(), func() {
+	f := framework.NewDefaultFramework("apiserver")
 
 	When("the manifest changes", func() {
 		BeforeEach(func() {
-			cfg, _ = clientcmd.BuildConfigFromFlags("", framework.TestContext.KubeConfig)
```

Comment:
for familiarity: is this not needed because `f := framework.NewDefaultFramework("apiserver")` is added above?

Reply:
yes! I'm not well versed in the testing framework but upstream has made several ergonomic improvements so now the config and client set can be extracted from the framework itself